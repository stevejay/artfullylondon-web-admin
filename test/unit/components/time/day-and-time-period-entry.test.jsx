import React from 'react'
import { shallow } from 'enzyme'
import _ from 'lodash'

import DayAndTimePeriodEntry
  from '_src/components/time/day-and-time-period-entry'

it('should render correctly', () => {
  const wrapper = shallow(
    <DayAndTimePeriodEntry
      value={{
        key: 'some-key',
        day: '2',
        from: '13:00',
        to: '14:00',
        timesRangeId: 'some-id',
        audienceTags: [{ id: 'tag-id', label: 'Tag Name' }]
      }}
      onDelete={_.noop}
    />
  )

  expect(wrapper).toMatchSnapshot()
})
