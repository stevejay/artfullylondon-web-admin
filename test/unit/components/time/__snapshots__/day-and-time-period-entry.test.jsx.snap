// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<TimeGridRow
  audienceTags={
    Array [
      Object {
        "id": "tag-id",
        "label": "Tag Name",
      },
    ]
  }
  day="Wednesday"
  keyId="some-key"
  onDelete={[Function]}
  time="13:00 to 14:00"
  timesRange="some-id"
/>
`;
